backend/internal/system/cmodels/property.go

```diff
@@ -58,6 +63,25 @@ func NewProperty(name, value string, isSecret bool) (*Property, error) {
 	return property, nil
 }
 
+// NewMultiValuedProperty creates a new multi-valued Property instance.
+// If isSecret is true, all values will be automatically encrypted.
+func NewMultiValuedProperty(name string, values []string, isSecret bool) (*Property, error) {
```

Comment:
I'd suggest to change the existing `NewProperty` method to accept the multi valued parameter rather than introducing a new method